### DISCUSSION TITLE
2.2.0

### DISCUSSION THREAD

**Discussion by PoeppingT** [awslabs/aws-saas-boost#482] [Announcements]
Hello curious reader! 🚀 This release marks the official v2.2.0 release of SaaS Boost, now with support for S3 buckets as part of your Application definitions. This release contains that feature along with various bugfixes and security improvements over the previous release.

A reminder that you can view our project roadmap by viewing the "Projects" page on our GitHub repository page. Please upvote and comment on issues you'd like to see, and don't be afraid to open discussions to ask questions.

A full changelist is below. Thanks for reading!

## What's Changed
* Manually signout in Cognito environments with a /logout redirect. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/448
* Fix apigateway permissions in Onboarding for CloudFormation by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/449
* Fix for Issue #452 by @raman231 in https://github.com/awslabs/aws-saas-boost/pull/458
* Fix for Issue #446 by @raman231 in https://github.com/awslabs/aws-saas-boost/pull/457
* Tenant update race condition by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/451
* Remove deprecated Windows 20H2 option. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/459
* Rds options orderable instance fix by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/460
* Fix billing dropdown selection and return Stripe plans from billing service. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/411
* Fix Stripe Billing Setup by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/462
* SaaS Boost API access via OAuth client credentials grant by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/450
* Remove unused test Stripe keys from billing service by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/464
* Support S3 storage for tenants as AppConfig extension. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/463
* update iam endpoint setting for gcr by @goodbyedavid in https://github.com/awslabs/aws-saas-boost/pull/468
* Pass S3 extension bucket name through AppConfig rather than as an SSM… by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/467
* add ningxia ELB account id by @goodbyedavid in https://github.com/awslabs/aws-saas-boost/pull/473
* Change "Users" to "System Users" in UI. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/475
* Improve Application Config UX by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/477
* Fix line graphs in metrics dashboards by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/478
* Update scripts must not fail if Lambda function does not already exist. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/480

**Full Changelog**: https://github.com/awslabs/aws-saas-boost/compare/v2.1.0...v2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/awslabs/aws-saas-boost/releases/tag/v2.2.0'>2.2.0</a>.</em>

**Comment by rshettyami** [comment]
@PoeppingT  I am running the latest from main that I pulled last week 2/21/23, I have following question

1. Does this version support pool model vs silo model that I see, I don't see the pool model, please confirm.
2. Also I don't see the "Billing Plan" selection, below text is from your use guide https://github.com/awslabs/aws-saas-boost/blob/main/docs/user-guide.md

Billing Plan - As your tenants are onboarded, associate them with a specific billing plan. AWS SaaS Boost has a preconfigured range of billing plans. Ultimately, you'll want to better align these billing plan with the options that best align with your business model. This is achieved by configuring your products in Stripe and mapping them to the plans that are managed within AWS SaaS Boost.

> 
![image](https://user-images.githubusercontent.com/126362707/221860634-5a5a37c9-c28f-463e-9360-00bea8497331.png)

**Reply by PoeppingT** [reply]
Hi @rshettyami 
1. Pool model is not yet supported, but is on our backlog as a feature request https://github.com/awslabs/aws-saas-boost/issues/212
2. The billing plan section is only shown on the tenant onboarding screen if your AppConfig has billing enabled and an API key configured.

**Comment by daniel-hutao** [comment]
#492
